examples/with-typescript/pages/secure.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <b-alert show variant="warning">This is a secure page!</b-alert>
+    <b-row>
+      <b-col md="8">
+        <b-card title="State">
+          <pre>{{ state }}</pre>
+        </b-card>
+      </b-col>
+      <b-col md="4">
+        <b-card title="Scopes" class="mb-2">
+          User:
+          <b-badge>{{ $auth.hasScope('user') }}</b-badge>Test:
+          <b-badge>{{ $auth.hasScope('test') }}</b-badge>Admin:
+          <b-badge>{{ $auth.hasScope('admin') }}</b-badge>
+        </b-card>
+        <b-card title="token">{{ $auth.token || '-' }}</b-card>
```

Comment:
Since we added Token class, use `$auth.token` here will cause an error preventing this page from loading.

Can you change to `$auth.token.get() || '-' `?